### PR TITLE
Form verification in javascript created

### PR DESCRIPTION
**Pull request by ghost**
Form verification working to check for capitals in emails.

### CODE REVIEW

File: contact.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  if (!isEmail(emailValue)) {
+    setErrorFor(email, 'Can not submit email with uppercase characters');
+  } else { form.submit(); }
+}
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  checkInputs();
+});
```
Comment (mateo951) [inline contact.js]: There is an issue with email validation at the moment. If the uppercase is before the @, the error message is displayed correctly and the form is not submitted. Now, if the uppercase is after the @, it's being handled as a false negative and the form is submitted 😥 Kindly make sure that the validations are working on both cases 🚀 You got this!

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi Team, 

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: contact.js (hunk 1 above)
Comment (ghost) [reply contact.js]: Thank you for noticing! Already made the correction.

**Review by DammyShittu — APPROVED** [review]
## Hi Team 

 Amazing implementation of the suggestion that was given by the previous reviewer.

The validation is now working as it should ✔️ 

# APPROVED 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**